config/materials/iso1043_abs.mtl

```diff
@@ -31,7 +31,7 @@ PARAMETERS =
   Name = PTC_MATERIAL_DESCRIPTION
   Type = String
   Default = 'ABS for CNC machining
-Source: datasheet from supplier '
+Source: datasheet from supplier'
```

Comment:
@Mick3Lozzo I would just add "Momotek" before supplier

Reply:
> @Mick3Lozzo I would just add "Momotek" before supplier

I don't know if we have enough digits. I'll try.